components/menu/README.md

```diff
@@ -104,14 +96,6 @@ The `d2l-menu-item` component is used with JS handlers and can be wired-up to th
 * `supporting`: Allows supporting information to be displayed on the right-most side of the menu item
 <!-- docs: end hidden content -->
 
-### Accessibility Properties
-
-To make your usage of `d2l-menu-item` accessible, use the following property:
-
-| Attribute | Description |
-|--|--|
-| `description` | Overrides aria-label to provide extra information to screen reader users |
```

Comment:
Since this isn't listed in the `Properties` table above it probably should be added (same with the other cases below)